### PR TITLE
Executive Copy - 2022-04-15

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: governance/votes/Executive vote - April 15, 2022.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Template - [Executive Vote] Offload TUSD, Curve ETH-stETH Vault Parameter Changes, DAI Transfers - April 15, 2022
+summary: Offload TUSD collateral via liquidation, increase CRVV1ETHSTETH-A Maximum Debt Ceiling and Target Available Debt, DAI Transfers to GRO-001, Ambassadors Program, Gelato Keepers.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - April 15, 2022.md:3]: Should Ambassador be singular or plural? Kat used both in her forum thread. Personally I feel Ambassador Program scans better

File: governance/votes/Executive vote - April 15, 2022.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+## Proposal Details
+
+### TUSD Collateral Offloading
+
+As per [this](https://vote.makerdao.com/polling/QmVkRdjg#poll-detail) successful governance poll, TUSD collateral in the Maker Protocol will be liquidated under the parameters below if this executive proposal passes. The communication and lead-time requirements for offboarding collateral defined in [MIP62](https://mips.makerdao.com/mips/details/MIP62) have been met. 
+
+* Enable liquidations for TUSD-A
+* Auction Price Function: Linear Decrease
+	* tau: 250 days (estimated 10bps drop per 6 hours = 250 days till 0)
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - April 15, 2022.md:44]: Is there anything to link to for `tau`?

File: governance/votes/Executive vote - April 15, 2022.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+* Increase [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 5 million DAI to 50 million DAI.
+* Increase [Target Available Debt](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) from 3 million DAI to 10 million DAI.
+
+For further information, please see the proposal forum thread [here](https://forum.makerdao.com/t/immediate-short-term-parameter-changes-proposal-for-crvv1ethsteth-a-dc-and-gap-increase/14476).
+
+### Growth Core Unit Budget
+
+As per the ratified [GRO-001 DAI Budget](https://mips.makerdao.com/mips/details/MIP40c3SP37#budget-implementation), **474,683 DAI** will be transferred to the [Growth Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4) if this executive proposal passes. This is the second of two lump-sum transfers specified in the budget proposal.
+
+### Fund Ambassador Program Pilot
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - April 15, 2022.md:67]: OK you've gone with Ambassador Program here :rofl:

File: governance/votes/Executive vote - April 15, 2022.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+As per [this](https://vote.makerdao.com/polling/QmPpQ49p#poll-detail) successful governance poll, **25,000 DAI** will be transferred to the Ambassadors Program multi-sig wallet if this executive proposal passes.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - April 15, 2022.md:69]: Include wallet address/etherscan link?

File: governance/votes/Executive vote - April 15, 2022.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+As per [this](https://vote.makerdao.com/polling/QmeTfgKP#poll-detail) successful governance poll, CRVV1ETHSTETH-A (Curve ETH-stETH pool) parameters will be changed as detailed below if this executive proposal passes.
+* Increase [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 5 million DAI to 50 million DAI.
+* Increase [Target Available Debt](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) from 3 million DAI to 10 million DAI.
+
+For further information, please see the proposal forum thread [here](https://forum.makerdao.com/t/immediate-short-term-parameter-changes-proposal-for-crvv1ethsteth-a-dc-and-gap-increase/14476).
+
+### Growth Core Unit Budget
+
+As per the ratified [GRO-001 DAI Budget](https://mips.makerdao.com/mips/details/MIP40c3SP37#budget-implementation), **474,683 DAI** will be transferred to the [Growth Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4) if this executive proposal passes. This is the second of two lump-sum transfers specified in the budget proposal.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - April 15, 2022.md:65]: Include wallet address/etherscan link?

File: governance/votes/Executive vote - April 15, 2022.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+As per [this](https://vote.makerdao.com/polling/QmPpQ49p#poll-detail) successful governance poll, **25,000 DAI** will be transferred to the Ambassadors Program multi-sig wallet if this executive proposal passes. 
+
+For further information, please see the signal request thread [here](https://forum.makerdao.com/t/signal-request-fund-ambassadors-program-pilot/13199).
+
+### Replace Gelato Keeper Top-Up Contract DAI Stream
+
+As per [this](https://forum.makerdao.com/t/update-to-the-gelato-keeper-network-top-up-contract/14524) forum post, the [Protocol Engineering Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7) have included a replacement for the Gelato Keeper stream in this executive proposal to resolve a bug related to token approvals. This replacement will take place if this executive proposal passes.
+* The existing stream will be revoked.
+* **1,000 DAI per day** will be streamed to the new contract for 6 months.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - April 15, 2022.md:77]: Include wallet address/etherscan link?

File: governance/votes/Executive vote - April 15, 2022.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+### Fund Ambassador Program Pilot
+
+As per [this](https://vote.makerdao.com/polling/QmPpQ49p#poll-detail) successful governance poll, **25,000 DAI** will be transferred to the Ambassadors Program multi-sig wallet if this executive proposal passes. 
+
+For further information, please see the signal request thread [here](https://forum.makerdao.com/t/signal-request-fund-ambassadors-program-pilot/13199).
+
+### Replace Gelato Keeper Top-Up Contract DAI Stream
+
+As per [this](https://forum.makerdao.com/t/update-to-the-gelato-keeper-network-top-up-contract/14524) forum post, the [Protocol Engineering Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7) have included a replacement for the Gelato Keeper stream in this executive proposal to resolve a bug related to token approvals. This replacement will take place if this executive proposal passes.
+* The existing stream will be revoked.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - April 15, 2022.md:76]: What's happening to funds that have been streamed already? Are Gelato working around that at their end?

File: governance/votes/Executive vote - April 15, 2022.md (hunk 2 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - April 15, 2022.md:44]: Nope :(

**Pull request merged**